### PR TITLE
Add support for manipulating individual table rows

### LINKED ISSUE #500: Option to remove rows or columns from a table
**Is your feature request related to a problem? Please describe.**
'm always frustrated when I want to remove and update field content or remove rows from a table, to use "Live". 

**Describe the solution you'd like**
Remove rows or columns, and update content in a specific field in the table. 

**Describe alternatives you've considered**
Right now, every time you need to create a new table, and recast the desired content out of it. I have no other alternative.

### PR DESCRIPTION
**Pull request by patriksvensson**
Closes #500

### CODE REVIEW

**Comment by patriksvensson** [comment]
I need to write an example for the code as well. That's why this is a draft PR.

@phil-scott-78 Once you have some spare cycles, could you take a look at it and give me your opinion about the API for this?

**Comment by patriksvensson** [comment]
@phil-scott-78 This one is ready for review whenever you have some spare time

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -89,7 +89,7 @@ jobs:
         shell: bash
         run: |
           dotnet tool restore
-          dotnet example --all
+          dotnet example --all --skip live --skip livetable --skip prompt
```
Comment (phil-scott-78) [inline .github/workflows/ci.yaml:92]: awesome!

File: src/Spectre.Console/Extensions/TableExtensions.cs [hunk 2]
```diff
@@ -143,6 +166,80 @@ public static Table AddRow(this Table table, params string[] columns)
...
+            return InsertRow(table, index, columns.Select(column => new Markup(column)));
+        }
+
+        /// <summary>
+        /// Removes a row from the table with the specified index.
+        /// </summary>
+        /// <param name="table">The table to add the row to.</param>
+        /// <param name="index">The index to remove the row at.</param>
+        /// <returns>The same instance so that multiple calls can be chained.</returns>
+        public static Table RemoveRow(this Table table, int index)
```
Comment (phil-scott-78) [inline src/Spectre.Console/Extensions/TableExtensions.cs:232]: Would it make sense to name this `RemoteRowAt` to keep with the `IList` naming pattern?

File: src/Spectre.Console/Widgets/Table/TableRowCollection.cs [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+                _list.Insert(index, row);
+                return _list.IndexOf(row);
+            }
+        }
+
+        /// <summary>
+        /// Removes a row at the specified index.
+        /// </summary>
+        /// <param name="index">The index to remove a row at.</param>
+        public void Remove(int index)
```
Comment (phil-scott-78) [inline src/Spectre.Console/Widgets/Table/TableRowCollection.cs:96]: Same `RemoveAt` comment from `TableExtensions.cs `

**Review by phil-scott-78 — COMMENTED** [review]
Everything looks great to me, demo runs well on my machine too (of course). Just the `RemoveAt` comment.

Also loving being able to skip certain demos with `dotnet example` now too~

File: src/Spectre.Console/Extensions/TableExtensions.cs (hunk 2 above)
Comment (patriksvensson) [reply src/Spectre.Console/Extensions/TableExtensions.cs:232]: You're absolutely right. Will change that.

File: src/Spectre.Console/Widgets/Table/TableRowCollection.cs (hunk 3 above)
Comment (patriksvensson) [reply src/Spectre.Console/Widgets/Table/TableRowCollection.cs:96]: You're absolutely right. Will change that.

**Pull request merged**